test/integration/integration_test.go

```diff
@@ -180,9 +193,23 @@ func TestMain(m *testing.M) {
 		os.Exit(1)
 	}
 
+	var providerFile string
+	if isScaleTest {
+		providerFile = "eks.tf"
```

Comment:
For the future, we might move the scale testing TF to its own directory, then we could just copy over the TF from it into the target TF working directory. That would allow is to extend the TF for a specific cluster target. Not a blocker.